README.md

```diff
@@ -1,68 +1,9 @@
-# Zulip Send Message Action
+# Zulip GitHub Actions
 
 [![GitHub Action badge](https://github.com/zulip/github-actions-zulip/workflows/test-local/badge.svg)](https://github.com/zulip/github-actions-zulip/actions?query=workflow%3Atest-local)
 
-This action sends a message to [Zulip](https://zulip.com/).
+This is a collection of GitHub Actions to interact with [Zulip](https://zulip.com/).
```

Comment:
The main README now states that this repository contains a collection of GitHub Actions.
In the "Actions" section, for each action, we have a link to the README.

Does that work for you?